webapp/components/post_view/components/post_list.jsx

```diff
@@ -139,6 +132,15 @@ export default class PostList extends React.Component {
             });
         }
 
+        // Postpone all DOM related calculations to next frame.
+        // scrollHeight etc might return wrong data at this point
```

Comment:
What do you mean by this comment? 

`// scrollHeight etc might return wrong data at this point`